content/GettingStarted/Install/InstallingSCALE.md

```diff
@@ -156,26 +161,27 @@ After following the steps to install, reboot the system and remove the install m
 {{< expand "Troubleshooting" "v">}}
 If the system does not boot into TrueNAS SCALE, there are several things you can check to resolve the situation:
 
-* Check the system BIOS and see if there is an option to change the **USB emulation** from **CD/DVD/floppy** to **hard drive**. 
+* Check the system BIOS and see if there is an option to change the **USB emulation** from **CD/DVD/floppy** to **hard drive**.
   If it still does not boot, check to see if the card/drive is UDMA compliant.
 * Check to see if the system BIOS supports **UEFI with BIOS emulation**, if not, see if it has an option to boot using **legacy BIOS mode**.
 
-* If the system starts to boot but hangs with this repeated error message: `run_interrupt_driven_hooks: still waiting after 60 seconds for xpt_config`, 
+* If the system starts to boot but hangs with this repeated error message: `run_interrupt_driven_hooks: still waiting after 60 seconds for xpt_config`,
   go into the system BIOS and look for an onboard device configuration for a `1394 Controller`. If present, disable that device and try booting again.
-* If the burned image fails to boot and the image was burned using a Windows system, wipe the USB stick before trying a second burn using a utility such as [Active@ KillDisk](https://www.killdisk.com/eraser.html). 
-  Otherwise, the second burn attempt fails as Windows does not understand the partition that was written from the image file. 
+* If the burned image fails to boot and the image was burned using a Windows system, wipe the USB stick before trying a second burn using a utility such as [Active@ KillDisk](https://www.killdisk.com/eraser.html).
+  Otherwise, the second burn attempt fails as Windows does not understand the partition that was written from the image file.
```

Comment:
too much past tense, so suggest this change to get it more active voice, present tense.
```suggestion
  Otherwise, the second burn attempt fails as Windows does not understand the partition written from the image file.
```